graylog2-web-interface/src/stores/streams/StreamRulesStore.ts

```diff
@@ -75,6 +76,9 @@ class StreamRulesStore {
   _emitChange() {
     this.callbacks.forEach((callback) => callback());
   }
+  unregister(callback: Callback) {
+    lodash.pull(this.callbacks, callback);
```

Comment:
I don't doubt `lodash` is useful, but so far we only use a single function of it. As far as I know, nobody in the team knows it and we don't even have it as a direct dependency.
I'm not against adding 24kb of dependency if we need it (we already have many others, so there's not much difference), but in this case I have the feeling we are using it for some small eye candy.